wire-schema/src/main/java/com/squareup/wire/schema/internal/Util.kt

```diff
@@ -78,7 +88,16 @@ object Util {
 
   /** True if the supplied value is in the valid tag range and not reserved.  */
   fun isValidTag(value: Int) =
-    value in MIN_TAG_VALUE until RESERVED_TAG_VALUE_START ||
-        value in (RESERVED_TAG_VALUE_END + 1) until MAX_TAG_VALUE + 1
+      value in MIN_TAG_VALUE until RESERVED_TAG_VALUE_START ||
+          value in (RESERVED_TAG_VALUE_END + 1) until MAX_TAG_VALUE + 1
+}
 
-}
\ No newline at end of file
+internal fun ProtoMember.isExtensionField(schema: Schema): Boolean {
```

Comment:
I don’t like this API. The ProtoMember is an identifier, not a value. It’d be like `"Canada".isCountry()`

Reply:
Would be better to define an API on `Schema`.